config/config.go

```diff
@@ -156,8 +156,9 @@ type GDPR struct {
 	Timeouts                GDPRTimeouts `mapstructure:"timeouts_ms"`
 	NonStandardPublishers   []string     `mapstructure:"non_standard_publishers,flow"`
 	NonStandardPublisherMap map[string]int
-	TCF2                    TCF2 `mapstructure:"tcf2"`
-	AMPException            bool `mapstructure:"amp_exception"`
+	TCF2                    TCF2     `mapstructure:"tcf2"`
+	AMPException            bool     `mapstructure:"amp_exception"`
+	CountryList             []string `mapstructure:"country_list"`
```

Comment:
Perhaps we can name it ` eea_countries` to match what's already in PBS-Java, or else something more descriptive such as `countries_to_force_enforcement`. Ok, that last one is a bit too long.. but I think you get the idea. "list" isn't descriptive.